### DISCUSSION TITLE
Solenoid 5/3-way Center Exhaust

### DISCUSSION THREAD

**Discussion by JewelBoner** [frc5567/2023-Robot#184] [Team Posts]
https://tameson.com/pneumatics/directional-valve/electrical-actuated/5-3-way/mvsc-220-4e2r-dc24-g-g1-4inch-24vdc-5-3-way-center-exhaust-solenoid-valve-1p5-8bar.html

**Comment by pangoliff** [comment]
Link to a table that lists a bunch more of these (Most if not all of them are more expensive, but I still thought it could be useful.)
https://www.festo.com/cat/en-us_us/data/doc_enus/PDF/US/ISO5599_ENUS.PDF
